server/src/parser/table-reader.ts

```diff
@@ -92,6 +92,23 @@ class LineReader {
     }
   }
 
+  private trimPipes(line: string) {
+    // in case empty first cell, do not remove space after leading pipe
+    if (line.startsWith("| | ")) {
+      line = line.substring(1, line.length);
+    }
+    // remove mandatory leading pipe
+    if (line.startsWith("| ")) {
+      line = line.substring(2, line.length);
+    }
+    // remove optional pipe at the end of line
+    if (line.endsWith(" |")) {
```

Comment:
Trimming the otional last pipe at the end of line is fine, as it doesn't affect the location information